spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -323,9 +323,46 @@ def get_rating_summary(app_id, platform, versionId = '')
       parse_response(r, 'data')
     end
 
+    def create_developer_response(app_id: nil, platform: nil, review_id: nil, response: "")
```

Comment:
`app_id`, `platform`, `review_id` and `response` are required.
So this interface is not good.
You don't pass the default value to arguments.

```ruby
def create_developer_response(app_id, platform, review_id, response)
```